modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/AbstractTableCommand.java

```diff
@@ -41,6 +42,10 @@ public abstract class AbstractTableCommand implements CatalogCommand {
     }
 
     private void validate() {
+        if (schemaName != null && CatalogUtils.isSystemSchema(schemaName)) {
+            throw new CatalogValidationException(format("Operations with reserved schemas "
+                    + "keywords are not allowed, schema: {}", schemaName));
```

Comment:
I think there is a typo here.

>Operations with reserved schemas keywords are not allowed, schema: 

The word `keywords` is unnecessary.

Reply:
done